source/less/global/fonts.less

```
@@ -0,0 +1,29 @@
+@font-face {
+  font-family: "Open Sans";
+  font-weight: 300;
+  font-style: normal;
+  font-display: swap;
+  src:
+    url("/fonts/opensanslight.woff2") format("woff2"),
```

Здесь тоже ошибка с относительными путями:
```suggestion
    url("../fonts/opensanslight.woff2") format("woff2"),
```
(и так далее с остальными шрифтами).